### PR TITLE
Show Command

### PR DESCRIPTION
**Pull request by marcbonnici**
Add show command to WA3

### CODE REVIEW

File: wa/commands/show.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        self.parser.add_argument('plugin', metavar='PLUGIN',
+                                 help='The name of the plugin to display documentation for.')
+
+    def execute(self, state, args):
+        name = args.plugin
+        rst_output = None
+
+        plugin = get_plugin(name)
+        if plugin:
+            rst_output = get_rst_from_extension(plugin)
```
Comment (setrofim) [inline wa/commands/show.py]: Should rename this function to `get_rst_from_plugin`

File: wa/commands/show.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            # Correctly format the tile of the man page
+            output = output.replace('.SH', '.TH', 1)
+            call('echo "{}" | man -l -'.format(escape_double_quotes(output)), shell=True)
+        else:
+            print rst_output
+
+def get_plugin(name):
+    for kind in pluginloader.kinds:
+        results = pluginloader.list_plugins(kind)
+        for result in results:
```
Comment (setrofim) [inline wa/commands/show.py]: Why iterate by `kind` here? Why not just `for result in pluginloader.list_plugins():` ?

File: wa/commands/show.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            if name == result.name:
+                return result
+            if hasattr(result, 'alias'):
+                for alias in result.alias:
+                    if name == alias:
+                        return result
+
+def get_target(name):
```
Comment (setrofim) [inline wa/commands/show.py]: For the sake of clarity, I would rename this to something like `get_target_description`

**Comment by marcbonnici** [comment]
Comments Addressed

**Pull request merged**